src/bin/pants.rs

```diff
@@ -240,6 +235,13 @@ fn write_package_output(
     Ok(())
 }
 
+fn get_purl_color(vulnerable: bool, val: String) -> StyledObject<String> {
```

Comment:
`get_purl_color` to me sounds like this would return the color I need to apply the style myself. Does `style_purl` read better to anyone else?

Reply:
I think so, I named this really late at night, great observation!

Reply:
In: 0ce99e2